### PR TITLE
Closes #158 | Add Dataloader Bloom-VIST

### LINKED ISSUE #158: Create dataset loader for Bloom-vist
Dataloader name: `bloom_vist/bloom_vist.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?bloom_vist

|     Dataset| bloom_vist |
|-------------|---|
| Description | BLOOM VIST is a visual storytelling of books that consists of 62 languages indigenous to SEA. This dataset is owned by Bloom, a free, open-source software developed by SIL International and associated with Bloom Library, app, and services. This dataset is released with the LICENSE family of Creative Commons (although each story datapoints has its licensing in more detail, e.g cc-by, cc-by-nc, cc-by-nd, cc-by-sa, cc-by-nc-nd, cc-by-nc-sa) |
| Subsets | abc, ahk, bfn, bjn, bkx, brb, brv, bya, bzi, ceb, cgc, cmo, ddg, dmg, dnw, dtp, enc, fil, hil, hro, idt, ilo, ind, jra, kak, khb, khm, kqr, krr, ksw, lhu, lsi, lwl, mdr, mgm, mhx, mkz, mry, msb, mya, nod, nxa, nxl, pag, pce, pea, pmf, psp, sea, sgd, sml, snl, tdt, tet, tha, tkd, tpu, war, wms, yet, yin, zlm |
| Languages | abc, ahk, bfn, bjn, bkx, brb, brv, bya, bzi, ceb, cgc, cmo, ddg, dmg, dnw, dtp, enc, fil, hil, hro, idt, ilo, ind, jra, kak, khb, khm, kqr, krr, ksw, lhu, lsi, lwl, mdr, mgm, mhx, mkz, mnw, mqj, mry, msb, mya, nod, nxa, nxl, pag, pce, pea, pmf, psp, sea, sgd, sml, snl, tdt, tet, tha, tkd, tpu, war, wms, yet, yin, zlm |
| Tasks | Visual Storytelling |
| License | Creative Commons license family (cc) |
| Homepage | https://huggingface.co/datasets/sil-ai/bloom-vist |
| HF URL | https://huggingface.co/datasets/sil-ai/bloom-vist |
| Paper URL | https://aclanthology.org/2022.emnlp-main.590 |

### PR DESCRIPTION
**Pull request by sabilmakbar**
Closes #158.

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/bloom_vist/bloom_vist.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Comment by sabilmakbar** [comment]
Test Suite Run logs over all 62 langs:
[test_log.log](https://github.com/SEACrowd/seacrowd-datahub/files/13761793/test_log.log)

File: seacrowd/sea_datasets/bloom_vist/bloom_vist.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    "tkd": "Tukudede",
+    "tpu": "Tampuan",
+    "war": "Waray-Waray",
+    "wms": "Wambon",
+    "yet": "Yetfa",
+    "yin": "Riang Lai",
+    "zlm": "Malay",
+}
+
+#it's a gated dataset, hence _LOCAL = True
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/bloom_vist/bloom_vist.py:105]:
```suggestion
# it's a gated dataset, hence _LOCAL = True
```

**Review by ljvmiranda921 — APPROVED** [review]
Just a small nit but LGTM!

**Comment by sabilmakbar** [comment]
Hi LJ, thanks for the comment! I'm modifying the dataset based on a similar PR in [this comment](https://github.com/SEACrowd/seacrowd-datahub/pull/198#discussion_r1440680994). Please have a look and see if this dataloader PR has already implemented that suggestion.

File: seacrowd/sea_datasets/bloom_vist/bloom_vist.py [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+    "tkd": "Tukudede",
+    "tpu": "Tampuan",
+    "war": "Waray-Waray",
+    "wms": "Wambon",
+    "yet": "Yetfa",
+    "yin": "Riang Lai",
+    "zlm": "Malay",
+}
+
+_LOCAL = False
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/bloom_vist/bloom_vist.py:104]: 👍 !

**Review by ljvmiranda921 — APPROVED** [review]
Thanks for the update! LGTM, let's just wait for @ryanignatius 's review

**Review by ryanignatius — APPROVED** [review]
Reviewed and tested, LGTM

**Pull request merged**